api/tests/integration/infrastructure/repositories/campaign-repository_test.js

```diff
@@ -370,28 +370,56 @@ describe('Integration | Repository | Campaign', () => {
         });
       });
 
-      context('when some campaigns creatorId match the the given creatorId searched', () => {
+      context('when some campaigns creator firstName match the the given creatorName searched', () => {
 
         let filter;
 
         beforeEach(() => {
-          const creatorId1 = databaseBuilder.factory.buildUser({}).id;
-          const creatorId2 = databaseBuilder.factory.buildUser({}).id;
+          const creator1 = databaseBuilder.factory.buildUser({});
+          const creator2 = databaseBuilder.factory.buildUser({});
 
-          filter = { creatorId: creatorId1 };
+          filter = { creatorName: creator1.firstName };
 
           _.each([
-            { name: 'Maths L1', creatorId: creatorId1 },
-            { name: 'Maths L2', creatorId: creatorId2 },
-            { name: 'Chimie', creatorId: creatorId1 },
+            { name: 'Maths L1', creatorId: creator1.id },
+            { name: 'Maths L2', creatorId: creator2.id },
+            { name: 'Chimie', creatorId: creator1.id },
           ], (campaign) => {
             databaseBuilder.factory.buildCampaign({ ...campaign, organizationId });
           });
 
           return databaseBuilder.commit();
         });
 
-        it('should return the campaign with right creatorId', async () => {
+        it('should return the campaign with right creator firstName', async () => {
```

Comment:
Faire court 
```suggestion
        it('should return matching campaigns', async () => {
```
Ou faire long mais être précis ?
```suggestion
        it('should return campaigns created by user whose first name is supplied', async () => {
```

Reply:
Ta remarque est valable, je m'étais fait la même réflexion mais j'ai simplement mis à jour les tests existants en changeant `creatorId` par `creatorName`. Sinon, je trouve que ta première proposition de nom est suffisante.